server/catgenome/src/main/java/com/epam/catgenome/manager/project/ProjectManager.java

```diff
@@ -388,7 +394,38 @@ public Project create(final Project project, Long parentId) {
                 dataItems.forEach(i -> fillFileTypeLists(i, newGeneFiles, newVcfFiles));
             }
         }
-
+        if (!newProject && !CollectionUtils.isEmpty(loadedProject.getNotes())) {
+            List<Long> existingNoteIds = loadedProject.getNotes().stream()
+                    .map(ProjectNote::getNoteId)
+                    .collect(Collectors.toList());
+            List<Long> newNoteIds = helpProject.getNotes() == null ? Collections.emptyList() :
+                    helpProject.getNotes().stream()
+                    .map(ProjectNote::getNoteId)
+                    .collect(Collectors.toList());
+            List<Long> notesToDelete = existingNoteIds.stream()
+                    .filter(o -> !newNoteIds.contains(o))
+                    .collect(Collectors.toList());
+            if (!notesToDelete.isEmpty()) {
+                final Filter filter = Filter.builder()
+                        .field("note_id")
+                        .operator("in")
+                        .value("(" + join(notesToDelete, ",") + ")")
+                        .build();
+                projectDao.deleteProjectNotes(Collections.singletonList(filter));
+            }
+        }
+        List<ProjectNote> notes = project.getNotes();
+        if (!CollectionUtils.isEmpty(notes)) {
+            List<ProjectNote> notesToAdd = notes.stream()
+                    .filter(n -> n.getNoteId() == 0)
```

Comment:
I would expect that to add a new note it should be specified like this:
```
"notes": [ 
  {
    "title": "Note",
    "content": "123123"
  }
]
```
But NPE is thrown in this case